ampersand-form-view.js

```diff
@@ -161,9 +196,22 @@ assign(FormView.prototype, Events, {
         fieldView.parent = this;
         fieldView.render();
         this.fieldContainerEl.appendChild(fieldView.el);
-    }
-});
+    },
+
+    getValue: function(name) {
+        var field = this.getField(name, true);
+        return field.value;
+    },
+
+    setValue: function(name, value) {
+        var field = this.getField(name, true);
+        field.setValue(value);
```

Comment:
Does this need a check to see if `field` has `setValue` (like in setValues on L98)?